tests/views/test_collection_exercise.py

```diff
@@ -13,106 +14,51 @@
 from config import TestingConfig
 from response_operations_ui.controllers import collection_instrument_controllers
 from response_operations_ui.exceptions.error_codes import ErrorCode
-from response_operations_ui.exceptions.exceptions import ExternalApiError
+from response_operations_ui.exceptions.exceptions import ApiError, ExternalApiError
 from response_operations_ui.views.collection_exercise import (
     CIR_ERROR_MESSAGES,
-    build_collection_exercise_details,
-    get_existing_sorted_nudge_events,
+    _build_collection_instruments_details,
+    get_collection_exercise_and_survey_details,
+    get_sample_summary,
     validate_file_extension_is_correct,
     validate_ru_specific_collection_instrument,
 )
 from tests.views import ViewTestCase
 from tests.views.test_admin import url_permission_url, url_sign_in_data
 
-ci_selector_id = "efa868fb-fb80-44c7-9f33-d6800a17c4da"
+fake_response = namedtuple("Response", "url status_code text")
+
 collection_exercise_event_id = "b4a36392-a21f-485b-9dc4-d151a8fcd565"
 collection_exercise_id = "14fb3e68-4dca-46db-bf49-04b84e07e77c"
 collection_instrument_id = "a32800c5-5dc1-459d-9932-0da6c21d2ed2"
-collection_instrument_id_2 = "5c5ca56f-8d15-426d-969a-9799d68d7971"
-collection_instrument_id_3 = "bc0b2cdf-754c-4ffd-bab2-e30bf177ec80"
-collection_instrument_id_4 = "912f8a05-8f9c-4d90-bff5-825f45775822"
-period = "000000"
+period = "221_201712"
 sample_summary_id = "1a11543f-eb19-41f5-825f-e41aca15e724"
 short_name = "MBS"
 survey_id = "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87"
 survey_ref = "141"
-cir_guid = "427d40e6-f54a-4512-a8ba-e4dea54ea3dc"
 form_type = "0001"
 
 project_root = os.path.dirname(os.path.dirname(__file__))
 
 collex_root = f"{project_root}/test_data/collection_exercise/collection_exercise_details"
-no_sample = collex_root + "_no_sample.json"
-failed_validation = collex_root + "_failedvalidation.json"
 collex_details = collex_root + ".json"
 
 """Load all the necessary test data"""
 with open(collex_details) as json_data:
     collection_exercise_details = json.load(json_data)
 
-with open(no_sample) as json_data:
```

Comment:
This are cumbersome and full of repetition. Big chunks of JSON to use one or 2 elements. I got rid where ever I could